bridge: remove check for vid_end

If range not specified, make vid_end same as vid.
This will prevent upstream `bridge vlan add` from failing with -EINVAL

diff --git a/net/bridge/br_netlink.c b/net/bridge/br_netlink.c
index 7cb55c6..4544c7b 100644
--- a/net/bridge/br_netlink.c
+++ b/net/bridge/br_netlink.c
@@ -337,7 +337,10 @@ static int br_afspec(struct net_bridge *br,
 		if (!vinfo->vid || vinfo->vid >= VLAN_VID_MASK)
 			return -EINVAL;
 
-		if (!vinfo->vid_end || vinfo->vid_end >= VLAN_VID_MASK)
+		if (!vinfo->vid_end)
+			vinfo->vid_end = vinfo->vid;
+
+		if (vinfo->vid_end >= VLAN_VID_MASK)
 			return -EINVAL;
 
 		switch (cmd) {
